core/server/master/src/main/java/alluxio/master/AlluxioMasterRestServiceHandler.java

```diff
@@ -506,7 +508,7 @@ private Capacity getCapacityInternal() {
     // free/used
     // spaces, those statistics can be gotten via other REST apis.
     String filesPinnedProperty =
-        MetricsSystem.getMasterMetricName(FileSystemMaster.Metrics.FILES_PINNED);
+        MetricsSystem.getMasterMetricName(DefaultFileSystemMaster.Metrics.FILES_PINNED);
```

Comment:
As talked offline, we put this as a separate task and I have a JIRA created to track it:
ALLUXIO-2808 - Define Metrics in Interface rather than implementation class